data/strikes.json

```diff
@@ -1,98 +1,4 @@
-{
```

Comment:
Looks like this open brace character was inadvertently removed and needs to be added back to make this valid JSON.